src/azure-cli-core/azure/cli/core/telemetry.py

```diff
@@ -519,6 +527,14 @@ def _get_config():
     return _session.application.config
 
 
+@decorators.suppress_all_exceptions()
+def _get_secrets_warning_config():
+    show_secrets_warning = _get_config().getboolean('clients', 'show_secrets_warning', fallback=None)
```

Comment:
`getboolean` can't take `None` as the `fallback`. The below error now appears in the `--debug` log of every command:

```
cli.azure.cli.core.decorators: Suppress exception:
Traceback (most recent call last):
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/__main__.py", line 62, in <module>
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/__main__.py", line 55, in <module>
SystemExit: 0

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/decorators.py", line 79, in _wrapped_func
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/telemetry.py", line 532, in _get_secrets_warning_config
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\knack/config.py", line 147, in getboolean
ValueError: Not a boolean: None
```

Reply:
already fixed in https://github.com/Azure/azure-cli/pull/28387